packages/messaging/src/ui/components/CenteredView/CenteredView.tsx

```diff
@@ -12,5 +12,6 @@ const styles = StyleSheet.create({
     flex: 1,
     justifyContent: "center",
     alignItems: "center",
+    gap: 16
```

Comment:
instead of applying this here, let just extend ViewProps on the CenteredView, and then take the style prop and apply it on line 5, then itll be more reusable for things in the future. Not sure why I didn't do that in the first place 😓